lib/modules/collapseBooleanAttributes.es6

```diff
@@ -104,20 +104,24 @@ const amphtmlBooleanAttributes = new Set([
 
 
 export default function collapseBooleanAttributes(tree, options, moduleOptions) {
-    tree.match({attrs: true}, node => {
-        for (const attrName of Object.keys(node.attrs)) {
-            if (!node.tag) {
-                continue;
-            }
+    tree.walk(node => {
+        if (! node.attrs) {
+            return node;
+        }
```

Comment:
`tree.match`'s implementation is a little bit complex. Use `tree.walk` for simple matching will have better performance.